src/ape_geth/query.py

```diff
@@ -29,7 +39,18 @@ def estimate_contract_creation_query(self, query: ContractCreationQuery) -> Opti
         return None
 
     @perform_query.register
-    def get_contract_creation_receipt(self, query: ContractCreationQuery) -> Iterator[ReceiptAPI]:
+    def get_contract_creation_receipt(
+        self, query: ContractCreationQuery
+    ) -> Iterator[ContractCreation]:
         if self.network_manager.active_provider and isinstance(self.provider, EthereumNodeProvider):
-            if receipt := self.provider._get_contract_creation_receipt(query.contract):
-                yield receipt
+            ots = self.provider._make_request("ots_getContractCreator", [query.contract])
+            if ots is None:
+                return None
```

Comment:
Estimate query returning None is how we say that data is not available upstream of the query system

This rate can just do the call and see if it works, kind of a shortfall of the method (I am not sure of this query makes as much sense using the query system anymore)

Reply:
i think it makes more sense to skip on estimate is when the provider doesn't support ots namespace. we can check the provider once via ots_getApiLevel once we refactor into a separate otterscan provider.

here None is a final answer, as in "this is not a contract". it can still be expensive to put into estimate.

Reply:
addressed in https://github.com/ApeWorX/ape/pull/2001/commits/a070592625a287fc1abbbd8b7a4a74e295763059